schemas/answers/dropdown.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "answer": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "q_code": {
+        "$ref": "../common_definitions.json#/q_code"
+      },
+      "guidance": {
+        "$ref": "../common_definitions.json#/guidance"
+      },
+      "type": {
+        "type": "string",
+        "enum": [
+          "Dropdown"
+        ]
+      },
+      "mandatory": {
+        "type": "boolean"
+      },
+      "label": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "default_value": {
```

Comment:
Do we use this value in runner?

Reply:
My naming wasn't probably great but default_value is referring to the default drop-down option eg "Please select an answer".